### PR TITLE
Feature/animated btn

### PR DESCRIPTION
**Pull request by Upendra2003**

### CODE REVIEW

**Review by aayush105 — COMMENTED** [review]
@Upendra2003  add the image in README.md file as well, firstly update your branch and then push it inorder to overcome the conflict in README.md file.

**Comment by Upendra2003** [comment]
Updated the readme file

**Comment by Upendra2003** [comment]
can u merge it

**Comment by aayush105** [comment]
> can u merge it

There is a conflict again please resolve it and push it again.

**Comment by Upendra2003** [comment]
resolved !!

File: README.md [hunk 1]
```diff
@@ -74,10 +74,14 @@ Please be aware that the demos may exhibit significant accessibility issues, suc
...
   - [Scary animation](#scary-animation)
+  - [ANimated Button](#animated-btn)
+
+  - [Contributors](#contributors)
+  - [Contributing](#contributing)
   - [Light Bulb Animation](#light-bulb-animation)
   - [growing-flower](#growing-flower)
-- [Contributors](#contributors)
-- [Contributing](#contributing)
+  - [Contributors](#contributors)
```
Comment (aayush105) [inline README.md:83]: @Upendra2003 this portion is twice, fix and push it again. Remove the line no "79 and 80" and add your quick link above the "83 and 84" line.

**Review by Upendra2003 — COMMENTED** [review]
Done

**Review by aayush105 — APPROVED** [review]
LGTM!
Thank you @Upendra2003 for contributing.

**Pull request merged**